ansible/roles/ocp-workload-dm7-realtime-event-decisioning/tasks/wait_for_deploy.yml

```diff
@@ -6,25 +6,34 @@
 #
 # Manual Test to determine list of unready replication controllers :
 #  1) install jp :  https://github.com/jmespath/jp
-#  2) oc get rc -o json | jp 'items[?  (status.readyReplicas == ""|| status.readyReplicas == `0`) ].metadata.annotations."openshift.io/deployment-config.name"'
-#
+#  2) oc get rc -o json | jp 'items[?
+#    (status.readyReplicas == ""|| status.readyReplicas == `0`)
+#    ].metadata.annotations."openshift.io/deployment-config.name"'
+
 
 - name: "Wait for following replication controllers to be created: {{ item }}"
   command: 'oc get rc -o json -n "{{ ocp_project }}"'
   register: rc_state
   changed_when: false
   retries: "{{ deploy_status_retries }}"
   delay: "{{ deploy_status_delay }}"
-  until: 'rc_state.stdout |from_json | json_query("items[? (kind == ''ReplicationController'')].metadata.annotations.\"openshift.io/deployment-config.name\"") |intersect(item) |length >= 1'
-  loop: "{{pod_to_wait}}"
+  until: >-
+    rc_state.stdout | from_json | json_query("items[?
+    (kind == ''ReplicationController'')].metadata.annotations."openshift.io/deployment-config.name"")
+    | intersect(item) |length >= 1
```

Comment:
This logic is hard to follow, it would be better if the json query were a separate var referenced in the filter.